liblll/Compiler.cpp

```diff
@@ -99,14 +99,15 @@ std::string dev::eth::compileLLLToAsm(std::string const& _src, bool _opt, std::v
 
 string dev::eth::parseLLL(string const& _src)
 {
-	sp::utree o;
 	try
 	{
+		sp::utree o;
 		parseTreeLLL(_src, o);
+		ostringstream ret;
+		debugOutAST(ret, o);
+		killBigints(o);
+		return ret.str();
 	}
 	catch (...) {}
```

Comment:
It is, but at the same time the rest of the class must be fixed as they are all broken in the same way.